packages/zoe/src/contractFacet.js

```diff
@@ -371,6 +371,7 @@ export function buildRootObject(_vatPowers) {
         const ignoringUpdater = harden({
           updateState: () => {},
           finish: () => {},
+          reject: () => {},
```

Comment:
This is kind of okay, since the offerTable doesn't currently call reject(). OTOH, wouldn't it be better to have this method in the ignoringUpdater for compatibility in case there's ever a reason to use that API in offerTable?  What if someone decides to copy the ignoringUpdater for use somewhere else?

In the typescript descriptions, dropping this would make it incompatible with notifier.

Reply:
Hi @Chris-Hibbert I don't understand your comment. This change includes the `reject` method in the `ignoringUpdater` in the same sense in which it already has `updateState` and `finish` methods. What are you suggesting?

Reply:
Hmm. I don't know what I was seeing at the time, but (as you apparently could tell) I thought reject was being dropped from contractFacet.   I have no objection since it doesn't look like that now.

Sorry for the confusion.